src/Indications.gd

```diff
@@ -481,7 +492,10 @@ func get_selection_context(popup_method: Callable) -> ContextPopup:
 					can_move_up = true
 				if not can_move_down and base_tid[-1] < parent_child_count - filtered_count:
 					can_move_down = true
-		
+		if context == SELECTION_CONTEXT.VIEWPORT:
+			btn_arr.append(Utils.create_btn(TranslationServer.translate("View In List"),
+					view_in_list.bind(selected_tids[0]), false, load("res://visual/icons/ViewInList.svg")))
```

Comment:
There's a bit of oddness in doing this with `selected_tids[0]` when you have a multiselection, but it's not that important. I'll squash this eventually.